src/Common/Storage/Commands.Storage/Blob/Cmdlet/NewAzureStorageBlobSasToken.cs

```diff
@@ -80,6 +81,12 @@ public string Policy
             ParameterSetName = BlobPipelineParameterSetWithPermision)]
         public string Permission { get; set; }
 
+        [Parameter(Mandatory = false, HelpMessage = "Protocol can be used in the request with this SAS token.")]
+        public SharedAccessProtocol Protocol { get; set; }
```

Comment:
This is not an appropriate parameter type - as a user, it is unclear to me how to create this type.  This needs to be a simple type, or a type that can be easily created from another cmdlet.  Preferrably a simple type.